sdk/program/src/message/sanitized.rs

```diff
@@ -167,9 +167,10 @@ impl SanitizedMessage {
 
     /// Returns the fee payer for the transaction
     pub fn fee_payer(&self) -> &Pubkey {
-        self.account_keys()
-            .get(0)
-            .expect("sanitized message always has non-program fee payer at index 0")
+        self.account_keys().get(0).expect(
+            "sanitized messages always have a fee payer at index 0 \
```

Comment:
realize the previous string mentioned the program aspect, but I fail to see how it's relevant to the expectation that there's at least one key - the fee-payer.